pkg/client/src/app/queries/applications.ts

```diff
@@ -113,13 +109,19 @@ export const useCreateApplicationMutation = (
 
 export const useDeleteApplicationMutation = (
   onSuccess: () => void,
-  onError: (err: AxiosError) => void
+  onError: (err: AxiosError) => void,
+  getApplicationAssessment: (id: number) => Assessment | undefined
 ) => {
   const queryClient = useQueryClient();
-  return useMutation(deleteApplication, {
-    onSuccess: () => {
+  const { mutate: mutateAssessments } = useDeleteAssessmentMutation();
+  return useMutation(({ id }: { id: number }) => deleteApplication(id), {
+    onSuccess: (_, { id }) => {
       onSuccess();
       queryClient.invalidateQueries(ApplicationsQueryKey);
+      const assessment = getApplicationAssessment(id);
```

Comment:
@ibolton336 When I try to test this, I'm getting an error on this line when I delete an application:
```
TypeError: getApplicationAssessment is not a function
    at Object.onSuccess (applications.ts?33ea:121:26)
    at eval (mutation.js?832d:92:1)
```

Reply:
Looks like i only fixed it for the assessments page & this function is reused in the analysis page also. Fixing @mturley